tests/integration-tests/tests/update/test_update.py

```diff
@@ -110,16 +162,43 @@ def _test_update_compute_instance_type(region, cluster, new_compute_instance):
         max_monitoring_time=minutes(estimated_scaleup_time),
         number_of_nodes=number_of_nodes,
     )
-    _test_compute_instance_type(region, cluster.cfn_name, new_compute_instance)
+
+    return [node for node in slurm_commands.get_compute_nodes() if node not in initial_compute_nodes]
 
 
-def _test_compute_instance_type(region, stack_name, compute_instance_type):
+def _test_compute_instance_type(region, stack_name, compute_instance_type, host, must_fail=False):
```

Comment:
must_fail -> expect_to_fail?
same for the other function

Reply:
Removed since now I'm matching the previous value.